packages/apputils/src/licenses.tsx

```diff
@@ -309,10 +309,18 @@ export namespace Licenses {
     }
 
     /**
-     * Get the link to download the licenses in a given format.
+     * Download the licenses in the requested format.
      */
-    async getDownloadLink(options: IDownloadOptions): Promise<string> {
-      return `${this._licensesUrl}?format=${options.format}&download=1`;
+    async download(options: IDownloadOptions): Promise<void> {
```

Comment:
Taking the opportunity to make this `download` method more generic, to give more flexibility to downstreams that may want to override it.

Reply:
For now using the same approach and creating the `<a>` element is probably the simplest.

But it gives the possibility to create a `Blob` and set a custom file name.